module/modmanager/manager.go

```diff
@@ -581,10 +588,20 @@ func (m *module) startProcess(
 	if err := modlib.CheckSocketAddressLength(m.addr); err != nil {
 		return err
 	}
+
+	// Start module process with -debug if:
+	//  * debug is set to true on the module
+	//  * debug is unset on the module but manager log level is Debug
+	args := []string{m.addr}
+	if (m.debug != nil && *m.debug) ||
+		(m.debug == nil && logger.Level() == zapcore.DebugLevel) {
+		args = append(args, "-debug")
```

Comment:
I could also use `--debug`. IIUC the Go standard is a single dash, but other languages like Python seem to prefer double dashes.

Reply:
POSIX standard is that short dash is for single character options, double-dash is for word-like (long) options. If you stick with the single dash, some option parser may only see it as "-d" (but that can work if they're written with -d as a shorthand for --debug) At the end we just need to make sure our own SDKs follow the same convention. (So fine as-is, since this IS go after all.)